common/src/Microsoft.Azure.IIoT.Abstractions/src/Exceptions/MethodCallStatusException.cs

```diff
@@ -39,7 +39,7 @@ public class MethodCallStatusException : MethodCallException {
         /// <param name="errorMessage"></param>
         public MethodCallStatusException(string responsePayload, int result,
             string errorMessage = null) :
-            base($"Response {result} {errorMessage ?? ""}: {responsePayload}") {
+            base($"{{\"Message\":\"Response {result} {errorMessage ?? ""}\",\"Description\":{responsePayload}}}") {
```

Comment:
[Microsoft REST API Guidelines](https://github.com/Microsoft/api-guidelines/blob/master/Guidelines.md#710-response-formats) recommend to use `Details` together with `Message`. So maybe we should rename `Description` to `Details` ?

A few more points from there:
* `Details` there is also a list of objects instead of a single object, but I think that we can keep our single object here.
* The top level object also contains `error` key with the whole object that we assemble here as its value. In our case the object will become the value for `payload` key. So I am happy with that part of our existing structure as well.

What do you think, @dacolgit, @cristipogacean ?

Reply:
I'm ok for renaming `Description` to `Details`
@cristipogacean ?